packages/editor/src/components/post-view-link/index.js

```diff
@@ -6,28 +6,29 @@ import { Button } from '@wordpress/components';
 import { external } from '@wordpress/icons';
 import { store as coreStore } from '@wordpress/core-data';
 import { useSelect } from '@wordpress/data';
+import { store as preferencesStore } from '@wordpress/preferences';
 
 /**
  * Internal dependencies
  */
 import { store as editorStore } from '../../store';
 
-export default function PostViewLink( { showIconLabels } ) {
-	const { hasLoaded, permalink, isPublished, label } = useSelect(
-		( select ) => {
+export default function PostViewLink() {
+	const { hasLoaded, permalink, isPublished, label, showIconLabels } =
+		useSelect( ( select ) => {
```

Comment:
It's really interesting to see in how many components we were passing this prop down..

Reply:
yep, good cleanup